contentcuration/contentcuration/viewsets/invitation.py

```diff
@@ -4,22 +4,19 @@
 from rest_framework import serializers
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
-from rest_framework.status import HTTP_204_NO_CONTENT
 
 from contentcuration.models import Channel
 from contentcuration.models import Invitation
 from contentcuration.viewsets.base import BulkListSerializer
 from contentcuration.viewsets.base import BulkModelSerializer
 from contentcuration.viewsets.base import ValuesViewset
 from contentcuration.viewsets.common import UserFilteredPrimaryKeyRelatedField
-from contentcuration.viewsets.sync.constants import INVITATION
-from contentcuration.viewsets.sync.utils import add_event_for_user
-from contentcuration.viewsets.sync.utils import generate_update_event
 
 
 class InvitationSerializer(BulkModelSerializer):
-    accepted = serializers.BooleanField(default=False)
-    declined = serializers.BooleanField(default=False)
+    revoked = serializers.BooleanField(read_only=True)
```

Comment:
Will this make it so these fields can't be updated?

Reply:
Those are the default `read_only` values, but they get modified in the serializer's `get_fields` method [here](https://github.com/learningequality/studio/pull/2529/files/4db738520fdc2101b12a6e02a1b044ff4eb6da2e#diff-a88f6dbcd080c52d0d86750c15e9db0d5ee268c1079430af01a1c8c1046bde70R55-R67) based on who's logged on.  Unfortunately, this was the most straightforward way I could find to do per-field permissions, although once we upgrade to Django 3 there might be some better options available.

Reply:
(I'll add a comment to explain that in the code!)

Reply:
Ah sounds good!